packages/pg/lib/utils.js

```diff
@@ -23,8 +23,17 @@ function arrayString(val) {
       result = result + 'NULL'
     } else if (Array.isArray(val[i])) {
       result = result + arrayString(val[i])
-    } else if (val[i] instanceof Buffer) {
-      result += '\\\\x' + val[i].toString('hex')
+    } else if (ArrayBuffer.isView(val[i])) {
+      var item = val[i]
+      if (!(item instanceof Buffer)) {
+        var buf = Buffer.from(item.buffer, item.byteOffset, item.byteLength)
+        if (buf.length === item.byteLength) {
```

Comment:
Is this for Node 4.x compatibility? A comment might be useful.

Reply:
Actually, pg requires Node 8.x as a minimum now.

Reply:
Good to know! This was copy/pasted from below, which specifically calls out the Node.js V4 requirement.

I didn't want to diverge from established practice. If you're saying it's safe to drop code supporting Node.js V4, perhaps that should be done separately?